### PR TITLE
data ingest notebook and requirements.txt

### PR DESCRIPTION
**Pull request by aphawkins1**

### CODE REVIEW

File: data/data-goes-here.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```
Comment (Andrewwango) [inline data/data-goes-here.html:1]: What is the purpose of all this?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,112 @@
+aiohttp==3.8.3
```
Comment (Andrewwango) [inline requirements.txt:1]: Did you create this with pip freeze? If so, it should look cleaner, like this:
```
beautifulsoup4==4.6.0
bleach==2.0.0
bs4==0.0.1
certifi==2017.7.27.1
chardet==3.0.4
colorama==0.3.9
decorator==4.1.2
entrypoints==0.2.3
html5lib==0.999999999
idna==2.6
ipykernel==4.6.1
```

File: README.md [hunk 3]
```diff
@@ -1 +1,32 @@
-# GroupA_ML
\ No newline at end of file
+# GroupA_ML
```
Comment (Andrewwango) [inline README.md:1]: Please can you remove the content here as it is already in your data ingest notebook - we don't like duplicate text/code :)

File: Data-ingest-notebook.ipynb [hunk 4]
```diff
@@ -0,0 +1,443 @@
...
+    "**Format**\n",
+    "\n",
+    "Each record contains the following fields:\n",
+    "- domain: source of sentences\n",
+    "- avg_score: the mean formality rating, which ranges from -3 to 3 where lower scores indicate less formal sentences\n",
+    "- sentence\n",
+    "\n",
+    "**Download**\n",
+    "\n",
+    "To enable this notebook to run, please download the following .csv files  from [here](https://huggingface.co/datasets/osyvokon/pavlick-formality-scores/tree/main) and save them in ./data (Hint: click on the file size):\n",
```
Comment (Andrewwango) [inline Data-ingest-notebook.ipynb:42]: I like the intro - just one thing: for ease of use for future devs, could you code in downloading the files so you don't have to do it manually? `pd.read_csv` should work with urls so it should be easy, then `df.to_csv` to save it locally.

File: Data-ingest-notebook.ipynb [hunk 5]
```diff
@@ -0,0 +1,443 @@
...
+   "source": [
+    "## Custom test cases - Kainos job specifications"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee324517",
+   "metadata": {},
+   "source": [
+    "### Formal test case:\n",
```
Comment (Andrewwango) [inline Data-ingest-notebook.ipynb:409]: I really like these: please could you save these to a text file, maybe in a separate folder in `data` called `sanity_check` or something? Then it's really easy to reload this data when you evaluate your model. You can either write to a file using `open(fn, "w")` or use the `%%writefile` magic: https://ipython.readthedocs.io/en/stable/interactive/magics.html#cellmagic-writefile

**Review by Andrewwango — CHANGES_REQUESTED** [review]
Looks good, got most components! Just some minor changes to make things smoother. One last suggestion: can you change your notebook names to something like: 01-data... and 02-... so it's clear which notebook corresponds to which ticket to which PR! :)

File: Data-ingest-notebook.ipynb (hunk 4 above)
Comment (aphawkins1) [reply Data-ingest-notebook.ipynb:42]: I tried this in the first instance and kept getting SSL errors due to zscaler, so switched to having to manually download the data :(

File: data/data-goes-here.html (hunk 1 above)
Comment (aphawkins1) [reply data/data-goes-here.html:1]: This was just a place holder to ensure the file structure remained. I see you used a .txt file in your repo. I'll use that instead as it's more elegant

**Review by hellofadude — APPROVED** [review]

**Pull request merged**